commit b125811348b41682211de036d8a75aa08ae38734 (refs/changes/13/1113/2)
Author: Alex Wilson <alex.wilson@joyent.com>
Date:   2016-12-14T07:16:29+00:00 (2 years, 10 months ago)
    
    PUBAPI-1352 update cueball and bump pool max
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Reviewed by: Brian Bennett <brian.bennett@joyent.com>

diff --git a/package.json b/package.json
index 2271aa1..c9ec288 100644
--- a/package.json
+++ b/package.json
@@ -21,7 +21,7 @@
         "nopt": "2.0.0",
         "restify": "4.3.0",
         "bunyan": "1.8.1",
-        "cueball": "1.1.4",
+        "cueball": "1.1.9",
         "kang": "1.1.0",
         "sdc-clients": "10.0.0",
         "ufds": "1.2.0",
diff --git a/sapi_manifests/cloudapi/template b/sapi_manifests/cloudapi/template
index 9e9f947..107f0d5 100644
--- a/sapi_manifests/cloudapi/template
+++ b/sapi_manifests/cloudapi/template
@@ -85,7 +85,7 @@
             "{{{papi_domain}}}"
         ],
         "spares": 4,
-        "maximum": 20,
+        "maximum": 100,
         "recovery": {
             "default": {
                 "timeout": 2000,
